### PR TITLE
typed: impl Default for TypedHandle<T> when T is not Default

### PR DESCRIPTION
**Pull request by ijackson**
Signed-off-by: Ian Jackson <ijackson@chiark.greenend.org.uk>

### CODE REVIEW

**Review by thomcc — CHANGES_REQUESTED** [review]
This removes Debug too, but other than that seems good.

**Comment by ijackson** [comment]
Thom Chiovoloni writes ("Re: [thomcc/handy] typed: impl Default for TypedHandle<T> when T is not Default (#3)"):
> @thomcc requested changes on this pull request.
> 
> This removes Debug too, but other than that seems good.

Wait, what ?  Let me check...

-- 
Ian Jackson <ijackson@chiark.greenend.org.uk>   These opinions are my own.  

Pronouns: they/he.  If I emailed you from @fyvzl.net or @evade.org.uk,
that is a private address which bypasses my fierce spamfilter.

**Comment by ijackson** [comment]
Now fixed that.  Sorry, not sure how I managed to botch that!

File: src/typed.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ use core::marker::PhantomData;
 /// `T`, which is not true for a naïve implementation of this, so it's provided
 /// even though I don't think it's that helpful for most usage (handle maps
 /// already detect this at runtime).
-#[derive(Clone, Default, Debug)]
+#[derive(Clone, Default)]
```
Comment (thomcc) [inline src/typed.rs]:
```suggestion
#[derive(Clone, Debug)]
```

**Comment by ijackson** [comment]
Oops, just got that wrong again.  Sorry for flail.

**Review by thomcc — APPROVED** [review]

**Comment by ijackson** [comment]
Thom Chiovoloni writes ("Re: [thomcc/handy] typed: impl Default for TypedHandle<T> when T is not Default (#3)"):
> @thomcc approved this pull request.

Thanks.  I blame coronabrain...

Regards,
Ian.

-- 
Ian Jackson <ijackson@chiark.greenend.org.uk>   These opinions are my own.  

Pronouns: they/he.  If I emailed you from @fyvzl.net or @evade.org.uk,
that is a private address which bypasses my fierce spamfilter.

**Comment by thomcc** [comment]
Published v0.1.4 with this.

**Pull request merged**